styles/base.css

```diff
@@ -1,9 +1,7 @@
-/*1% = 11.75px
-1 rem = 16px*/
-
 body {
-  background-color: #CAEBF2;
-  color: #c9383b;
+  background-image: url("../images/uw-fountain.jpg");
+  background-attachment: fixed;
```

Comment:
if you want to cover the full browser size with the background image you can set a background-size propery, http://www.w3schools.com/cssref/css3_pr_background-size.asp